### DISCUSSION TITLE
[Error] [omni.hydra] [UsdToMdl] Prim XXX parameter XXX: References an asset that can not be found:

### DISCUSSION THREAD

**Discussion by shengyin1224** [StanfordVL/BEHAVIOR-1K#1043] [Q&A] [answered]
Thanks for your wonderful work! I want to load task-related config file such as /**/omnigibson/data/og_dataset/scenes/Beechwood_0_garden/json/Beechwood_0_garden_task_make_a_milkshake_0_0_template.json**. I refer to the code of **/omnigibson/examples/action_primitives/wip_solve_behavior_task.py** and modify the task name.

The code I run is below: 
[code.txt](https://github.com/user-attachments/files/17990901/code.txt)

The log I get is below:
[log.txt](https://github.com/user-attachments/files/17990902/log.txt)

I found only when I use the one and only scene and task in the example, I can success. If I try other scenes or tasks, there will be the following errors:

`2024-12-03 09:51:17 [6,928ms] [Error] [omni.hydra] [UsdToMdl] Prim '/World/scene_0/log_90/Looks/log_rlhqcm_base_link/Shader' parameter 'normalmap_texture': References an asset that can not be found: 'materials/log-rlhqcm-base_link-normal.png'
2024-12-03 09:51:17 [6,928ms] [Error] [omni.hydra] [UsdToMdl] Prim '/World/scene_0/log_90/Looks/log_rlhqcm_base_link/Shader' parameter 'reflectionroughness_texture': References an asset that can not be found: 'materials/log-rlhqcm-base_link-roughness.png'`

And the following error:

`  File "/home/magic-4090/miniconda3/envs/omnigibson/lib/python3.10/site-packages/omnigibson/systems/micro_particle_system.py", line 1699, in clothify_mesh_prim
    import pymeshlab  # We import this here because it takes a few seconds to load.
  File "/home/magic-4090/miniconda3/envs/omnigibson/lib/python3.10/site-packages/pymeshlab/__init__.py", line 11, in <module>
    from .pmeshlab import *
ImportError: /home/magic-4090/miniconda3/envs/omnigibson/lib/python3.10/site-packages/pymeshlab/lib/libmeshlab-common.so: undefined symbol: _ZdlPvm, version Qt_5`

or

"ValueError: invalid literal for int() with base 10: '0.0'"

Actually, I have already download the dataset.

I will appreciate it so much If you can help me!

**Comment by cgokmen** [comment, marked as answer]
Sorry for the late response. The issue in the title is a spurious warning that is safe to ignore. The second issue re: pymeshlab stems from a library in your conda environment that includes an incorrect version of Qt that prevents pymeshlab from launching. This only affects cloth objects which pymeshlab is used for. I recommend reinstalling with a clean conda environment.
